src/AppComponents/ExampleMatcher/Filter/CsvTrackReader.cpp

```diff
@@ -54,7 +54,7 @@ bool CsvTrackReader::operator()(
         timeList.emplace_back(try_time.first);
         pointList.emplace_back(Types::Track::Point::Latitude{std::stod(data[1])}, Types::Track::Point::Longitude{std::stod(data[2])});
         headingList.emplace_back(std::stod(data[3]));
-        velocityList.emplace_back(std::stod(data[4]));
+        velocityList.emplace_back(std::stod(data[4]) / 3.6);
```

Comment:
Changing existing logic (which normally can be done only on major version updates) has to be considered carefully as this breaks all current usages.

In this case I would reject this as m/s is the SI unit for velocity and km/h is not.

Reply:
At which point do we convert then?

Reply:
Btw, @bibermann - we DO EXACTLY convert km/h into m/s (remember 10 m/s = 36 km/h)
and reach the SI unit.
So, I will accept. The question is actually, what we expect as input and how we can ensure that.

Reply:
Either fix your input data and provide m/s, not km/h, or add an additional reader which may read your format.

Reply:
Than we have to adapt the other `JsonTrackReader` which had this conversion already in it.

I am absolutely on board to require m/s. I will change the commit.

Reply:
Ping @bibermann @markusabel